cache/README.md

```diff
@@ -0,0 +1,67 @@
+# Cache
+
+## Cache object
+
+```json
+{
+  "type": "string",
+  "timestamp": "object"
+}
+```
+
+## **Requests**
+
+|          Route           |                       Description                       |
+| :----------------------: | :-----------------------------------------------------: |
+| [GET /cache](#get-cache) | Returns the cloudflare cache meta data of last 24 hours |
```

Comment:
Why is this only for 24 hrs? How would we set those expectations when a client uses the API?

Reply:
How about `/MEMBER_CACHE?beforeHour=24`?

Reply:
Interesting, this makes sense and makes the API more dynamic to input

Reply:
But this API then will have some constraints